build.gradle.kts

```
@@ -32,7 +32,7 @@ allOpen {
 }
 
 group = "com.ampnet"
-version = "0.14.5"
+version = "0.14.6"
```

This could potentially be problematic/breaking change, so we can track it easily.
```suggestion
version = "0.15.0"
```